tools/readme_generator/tests/test_make_readme.py

```diff
@@ -1,36 +1,39 @@
-import os
+#!/usr/bin/env python3
+
+import difflib
 import tempfile
 import subprocess
 
 from pathlib import Path
 
-CWD = Path(os.path.split(os.path.realpath(__file__))[0])
-COMMIT_ID = "8f788213b363a46a5b6faa8f844d86d4adac9446"
+CWD = Path(__file__).resolve().parent
+
+TEST_APP_NAME = "gotosocial_ynh"
+TEST_APP_REPO = "https://github.com/yunohost-apps/gotosocial_ynh"
+TEST_APP_COMMIT_ID = "8f788213b363a46a5b6faa8f844d86d4adac9446"
+
+def diff_files(file_a: Path, file_b: Path) -> bool:
```

Comment:
You do realize that you don't need all of that because pytest already does all the diff for you '-'?

For example here https://github.com/YunoHost/apps/actions/runs/8265771728/job/22612209551 and here https://github.com/YunoHost/apps/actions/runs/8258315731/job/22590328725

And the diff is actually better than the difflib one :sweat_smile:

Reply:
Uh that's really weird because locally i don't have a diff at all...

Reply:
Did you followed the documentation on how to run the tests?